libs/ui/package.json

```diff
@@ -124,9 +118,10 @@
     "stream-browserify": "^3.0.0",
     "styled-components": "^5.3.11",
     "tmp": "^0.2.1",
-    "ts-jest": "29.1.1",
     "util": "^0.12.4",
-    "vite": "4.5.2"
+    "vite": "4.5.2",
+    "vitest": "^2.1.8",
+    "vitest-styled-components": "^1.0.0"
```

Comment:
I didn't realize you got this working! Nice!

Reply:
Yep! For anything that didn't quite work identically I just disabled it with a comment.